src/ints/int10.cpp

```diff
@@ -337,11 +337,9 @@ static Bitu INT10_Handler(void) {
 			{   
 				if (!IS_VGA_ARCH) break;
 				LOG(LOG_INT10,LOG_WARN)("Function 12:Call %2X (select vertical resolution)",reg_bl);
-				if (svgaCard != SVGA_None) {
-					if (reg_al > 2) {
-						reg_al=0;		// invalid subfunction
-						break;
-					}
+				if (reg_al > 2) {
+					reg_al = 0;	// invalid VGA subfunction
+					break;
```

Comment:
Looks like a mechanical change; I have no idea why this would be needed for `vgaonly` only but not the other VGA cards.

Reply:
Yes; now applies to all (S)VGA cards.